text/0016-named-tracers.md

```diff
@@ -2,7 +2,7 @@
 
 **Status:** `approved`
 
-_Creating Tracers and Meters using a factory mechanism and naming those Tracers / Meters in accordance with the library that provides the instrumentation for those components._
+_Creating Tracers and Meters using a Registry mechanism and naming those Tracers / Meters in accordance with the library that provides the instrumentation for those components._
```

Comment:
I'm not entirely happy with both factory and registry (altough registry is definitely better). Also I think we should get rid of the verb "naming" here, since we said we want to abolish the "named tracers" terminology (how about we also rename the file, btw?). Also, what is "those" referring to in that senctence?  How about:

```suggestion
_Associate potentially different Tracers and Meters along with meta-information with each library that implements instrumentation by parametrizing the APIs that create Tracers and Meters._
```

Reply:
I think this change is still fine, even if we don't want to get rid of "named" terminology.

Reply:
What would you think of:

>Associate Tracers and Meters with the name and version of the library which reports telemetry data by parameterizing the API which the library uses to acquire the Tracer or Meter.